Allow setting EBS volume throughput

evan10s opened the pull request:
## Description of the change

Adds a variable so you can customize the throughput setting for the EBS volumes created for workers to use. In my testing, I found that some instances of workers becoming unresponsive were solved by increasing the volume's throughput (which was easy to spot since the volume's `Volume throughput exceeded check` metric in AWS was set to 1 just after the time when the worker became unresponsive).

As for the new `volume_throughput` variable being nullable, in my testing the effect is:
- `null` means throughput is unspecified and throughput isn't specified on the launch template (and there's no drift on terraform plans)
- Setting the variable to a value causes that value to be used on the launch template

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue);
- [x] New feature (non-breaking change that adds functionality);
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected);
- [ ] Documentation (a documentation or example fix not affecting the infrastructure managed by this module);

## Checklists

### Development

- [x] All necessary variables have been defined, with defaults if applicable;
- [x] The code is formatted properly;

### Code review

- [x] The module version is bumped accordingly;
- [ ] Spacelift tests are passing;
- [x] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached;
- [x] This pull request is no longer marked as "draft";
- [ ] Reviewers have been assigned;
- [ ] Changes have been reviewed by at least one other engineer;

jubranNassar commented:
Hey @evan10s - Thank you for the PR 🙏 . two things here 1- can you add this change to the `examples` so it gets tested in our automated tests? 2- please bump up the `module_version` in `.spacelift/config.yml` otherwise this will fail because the version already exists

peterdeme reviewed (approved):
lgtm

peterdeme commented:
Can you rebase it @evan10s?

evan10s commented:
> Can you rebase it @evan10s? 🙏

Fixed the merged conflict and also added the new test from this PR to spacelift.yaml, since it seems like we'd missed that step
